esquery.js

```diff
@@ -377,7 +444,10 @@ function subjects(selector, ancestor) {
     if (selector == null || typeof selector != 'object') { return []; }
     if (ancestor == null) { ancestor = selector; }
     const results = selector.subject ? [ancestor] : [];
-    for (const [p, sel] of Object.entries(selector)) {
+    const keys = Object.keys(selector);
+    for (let i = 0; i < keys.length; i++) {
```

Comment:
Style nit: please use pre-increment/decrement whenever the value is not used.

*Aside:* Sorry that there's a few style things I'm going to manually nitpick in this PR. We don't have a very strict linting ruleset since this project hasn't had too many contributors. PRs welcome lol.